packages/widgets/wage-view/src/components/WageView/WageView.js

```diff
@@ -0,0 +1,45 @@
+import PropTypes from 'prop-types'
+import {useEffect} from 'react'
+import {form} from 'tocco-app-extensions'
+import EntityBrowserApp from 'tocco-entity-browser/src/main'
+import {LoadMask} from 'tocco-ui'
+
+const WageView = ({loadReports, reports, allowCreate, reportIds, searchFilters, intl}) => {
+  useEffect(() => {
+    loadReports(reportIds)
+  }, [loadReports, reportIds])
+
+  const modifyFormDefinition = formDefinition => {
+    if (!allowCreate) {
+      formDefinition = form.removeCreate(formDefinition)
+    }
+
+    if (reports.length > 0) {
+      formDefinition = form.addReports(formDefinition, reports, intl)
+    }
+
+    return formDefinition
+  }
+
+  return (
+    <LoadMask required={[reports]}>
+      <EntityBrowserApp
+        entityName="Wage"
+        formBase="WageView"
+        searchFilters={searchFilters}
+        modifyFormDefinition={modifyFormDefinition}
```

Comment:
I think we need to pass in the `appContext` here. At leas the `embedType` is needed otherwise the wrong `theme` will be used